go.mod

```diff
@@ -5,11 +5,13 @@ go 1.13
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/gobuffalo/flect v0.2.1
+	github.com/golang/protobuf v1.3.2 // indirect
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.9.0
 	github.com/spf13/afero v1.2.2
 	github.com/spf13/cobra v0.0.7
 	github.com/spf13/pflag v1.0.5
+	golang.org/x/text v0.3.2 // indirect
```

Comment:
The //indirect modules here has no need to be added. Could you please revert it?